be/src/olap/rowset/segment_v2/segment_iterator.cpp

```diff
@@ -1210,6 +1227,31 @@ Status SegmentIterator::_apply_inverted_index() {
         }
     }
 
+    // support expr to evaluate inverted index
+    std::unordered_map<ColumnId, std::pair<vectorized::NameAndTypePair, InvertedIndexIterator*>>
+            iter_map;
+
+    for (auto col_id : _common_expr_columns) {
```

Comment:
how about those non-array expr be processed in this situation?

Reply:
just will return NotSupport